examples/default-volume.yaml

```diff
@@ -0,0 +1,9 @@
+targetConfigs:
```

Comment:
I think we removed this test last pr

Reply:
right, removed that, we don't allow configuring the volume name anymore